Tangem/App/ViewModels/WalletModel.swift

```diff
@@ -486,12 +486,12 @@ extension WalletModel {
         wallet.getShareString(for: wallet.addresses[index].value)
     }
 
-    func exploreURL(for index: Int) -> URL? {
+    func exploreURL(for index: Int, amountType: Amount.AmountType?) -> URL? {
```

Comment:
```suggestion
    func exploreURL(for index: Int, amountType: Amount.AmountType? = nil) -> URL? {
```
Давай может так, и везде `nil` можно убрать будет)

Reply:
Может лучше token и передавать?

Reply:
Да, лучше токен и правда

Reply:
Плюсую за токен, чем более конкретный набор данных передаем, тем лучше